action/jsonapi.go

```diff
@@ -0,0 +1,210 @@
+package action
+
+import (
+	"bufio"
```

Comment:
Did you format this file with gofmt / goimports? The import order looks random.

Reply:
Yeah, I missed that. gofmt and goimports applied to the two new files.